README.md

```diff
@@ -57,13 +57,41 @@ null                   -> {}
 
 ### `stringifyRequest`
 
-Makes a request pretty and stringifies it. Absolute paths are replaced with relative ones.
+Turns a request into a string that can be used inside a `require()` or `import()` call or an `import` statement.
+Use it instead of `JSON.stringify(...)` if you're generating code inside a loader.
 
-Use it instead of `JSON.stringify(...)` to build code of a `require(...)` call in a loader.
+This function:
 
-``` javascript
-loaderUtils.stringifyRequest(this, require.resolve("./test"));
-// = "../node_modules/some-loader/lib/test.js"
+- resolves module requests into relative requests
```

Comment:
module request = "module/path"
relative request = "./relative/path"

stringifyRequest doesn't resolve one into another. Remove this line.